src/core/server/base.js

```diff
@@ -91,6 +92,12 @@ export function getPageProps({ store, req, res, config }) {
     log.debug('No userAgent found in request headers.');
   }
 
+  if (req.get(REGION_CODE_HEADER)) {
```

Comment:
I guess it's fine but we could also use a variable to avoid two calls to `.get()`.